apps/api/src/server/applications/application/representations/representaions.service.js

```diff
@@ -18,7 +18,7 @@ export const getCaseRepresentations = async (caseId, pagination, filterAndSort)
  * @returns {Promise<Prisma.RepresentationSelect>}
  */
 export const getCaseRepresentation = async (caseId, repId) => {
```

Comment:
worth removing the param caseId entirely since it's not needed anymore

Reply:
Thanks @francescobrenci-AR, good call. I've removed it